tests/test_getinfo.py

```diff
@@ -0,0 +1,275 @@
+import generate_tempfiles
```

Comment:
remove test_getinfo.py from this branch to avoid merge conflicts.

on the findshift1 branch: `git rm test_getinfo.py`